bims/views/location_site.py

```diff
@@ -379,3 +380,62 @@ def post(self, request, *args, **kwargs):
             extra_tags='location_site_form'
         )
         return HttpResponseRedirect(reverse('location-site-form'))
+
+
+class NonValidatedSiteView(
+        UserPassesTestMixin,
+        LoginRequiredMixin,
+        ListView):
+
+    model = LocationSite
+    context_object_name = 'location_sites'
+    template_name = 'non_validated_location_site.html'
+    paginate_by = 10
+
+    def test_func(self):
+        return user_has_permission_to_validate(self.request.user)
+
+    def handle_no_permission(self):
+        messages.error(self.request, 'You don\'t have permission '
+                                     'to validate location site')
+        return super(NonValidatedSiteView, self).handle_no_permission()
+
+    def get_context_data(self, **kwargs):
+        filter_owner = self.request.GET.get('owner', None)
+        filter_site_code = self.request.GET.get('site_code', None)
+        filter_river_name = self.request.GET.get('river_name', None)
+        context = super(
+            NonValidatedSiteView, self).get_context_data(**kwargs)
+        context['custom_url'] = ''
+        if filter_site_code:
+            context['custom_url'] = \
```

Comment:
@Fanevanjanahary please use bracket for new lines as suggested in the https://www.python.org/dev/peps/pep-0008/#maximum-line-length